__tests__/controllers/auth.controller.spec.ts

```diff
@@ -29,6 +29,23 @@ describe('creating an account', () => {
       })
     );
   });
+
+  it('returns http code 400 whith no gender defined', async () => {
+    const userFields = {
+      email:'testemail@email.com',
+      password:'holahola'
+    }
+    const response = await request(app)
+      .post(`${API}/auth/signup`)
+      .send(userFields);
+    expect(response.status).toBe(400);
+    expect(response.body).toStrictEqual(
+      expect.objectContaining({
+        errMessage: expect.any(String),
```

Comment:
It would be awesome to test the message content. Not sure but I think it should be saying why it's returning 400, right? Otherwise, I can't be sure if this is failing due to gender missing or the password being too short.